Horion/Module/Modules/Twerk.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Twerk.h"
+
+Twerk::Twerk() : IModule(0x0, Category::MOVEMENT, "Crouch Spam") {
+	registerFloatSetting("delay", &this->delay, this->delay, 0.05, 10);
+}
+
+Twerk::~Twerk() {
+}
+
+const char* Twerk::getModuleName() {
+	return ("Twerk");
+}
+
+void Twerk::onTick(C_GameMode* gm) {
```

Comment:
This entire function can be simplified into 

```cpp
if((Odelay++ % (delay * 20)) == 0){
    g_Data.getClientInstance()->getMoveTurnInput()->isSneakDown = !gm->player->isSneaking();
}
```

if ur gonna add twerk, you gotta do it right smh

/s